thirdparty/download_thirdparty.sh

```diff
@@ -9,7 +9,7 @@ source $TP_DIR/versions.sh
 
 download_extract_and_cleanup() {
 	filename=$TP_DIR/$(basename "$1")
-	curl -#LC - "$1" -o $filename
+	curl -L -k "$1" -o $filename
```

Comment:
` -k, --insecure      Allow connections to SSL sites without certs (H)` not so happy about that. Is this really needed for conda? Can we e.g. do this via a flag?